fastplotlib/graphics/selectors/_linear_region.py

```diff
@@ -215,6 +219,25 @@ def __init__(
                 pick_write=True,
             ),
         )
+
+        line0_outer = pygfx.Line(
+            pygfx.Geometry(
+                # share buffer with inner line so they can both be managed together
+                positions=line0.geometry.positions
+            ),
```

Comment:
Can it not share the whole geometry?

Reply:
ah I never thought of that, for me I see geometry and all I think of is the storage buffer on the GPU